tests/testsuite/fix.rs

```diff
@@ -1791,3 +1791,58 @@ fn non_edition_lint_migration() {
     // Check that it made the edition migration.
     assert!(contents.contains("from_utf8(crate::foo::FOO)"));
 }
+
+// This test will break after rust-lang/rust#88514 being fixed.
```

Comment:
Could the test mostly just assert that nothing is fixed, which means that this wouldn't break when rust-lang/rust is updated?

Reply:
Sure! Updated. Thank for the review 😆